application/common/components/MfaBackendWebAuthn.php

```diff
@@ -0,0 +1,187 @@
+<?php
```

Comment:
Given the high-urgency nature of this effort, I'm hesitant to suggest this, but for my own understanding I'll go ahead. Would it work to make an interface (or trait) for all that's common between WebAuthn and U2F?

Reply:
I _think_ we'll actually remove all the u2f code before this is deployed. I can't think of any reason to keep it once we convert over to webauthn